marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow-utils.sjs

```diff
@@ -718,8 +718,8 @@ function addMetadataToContent(content, flowName, stepName, jobId) {
  * @param configCollections the collections to add to each content object based on step definition / flow / step / runtime config
  * @return An object consisting of two properties - "transaction" and "dateTime"
  */
-function writeContentArray(contentArray, databaseName, configCollections = []) {
-  const vars = {contentArray, configCollections};
+function writeContentArray(contentArray, databaseName, configCollections = [], targetCollectionsAdditivity = false) {
```

Comment:
'targetCollectionsAdditivity' shouldn't be a parameter to writeContentArray() method. It can be part of the context within a content object.